README.md

```diff
@@ -0,0 +1,14 @@
+# members
+System for handling club entries in czech orienteering.
+
+## Website
+The website is avaible at https://members.eob.cz/.

```

Comment:
Pridal bych i odkaz na github, to readme se s kodem muze dostat i mimo github, tak aby se dalo najit kde ma domov.